driver/pelican/pelican.go

```diff
@@ -64,6 +66,35 @@ type apiThermostat struct {
 	RunStatus       string  `xml:"runStatus"`
 }
 
+// Thermostat History Object API Result Structs
+type apiResultHistory struct {
+	XMLName xml.Name   `xml:"result"`
+	Success int        `xml:"success"`
+	Message string     `xml:"message"`
+	Records apiRecords `xml:"ThermostatHistory"`
+}
+
+type apiRecords struct {
+	Name    string       `xml:"name"`
+	History []apiHistory `xml:"History"`
+}
+
+type apiHistory struct {
+	TimeStamp string `xml:"timestamp"`
+}
+
+// Object API Result Structs
```

Comment:
`Object` is pretty vague. The type of the object you're actually retrieving here is called `Site` by the Pelican documentation, just like we had `Thermostat` and `ThermostatHistory` before.

You should update the comment and type names so they're more clear.